src/oc/hoc_init.cpp

```diff
@@ -276,7 +278,7 @@ void hoc_init(void)	/* install constants and built-ins table */
 	int i;
 	Symbol *s;
 
-	_nrnunit_use_legacy_ = 0; /* default */
+	_nrnunit_use_legacy_ = 1; /* default */
```

Comment:
This seems like a non-trivial change!

Reply:
@alexsavulescu doesn't this change the behaviour of the code base?

Reply:
Yes, but it is temporary (as per commit message) in order to have Simulation Stack green as grass. Will get reverted just before merge - otherwise Travis fails

Reply:
Ok, thanks! Just went through the files by themselves, did not see that message.